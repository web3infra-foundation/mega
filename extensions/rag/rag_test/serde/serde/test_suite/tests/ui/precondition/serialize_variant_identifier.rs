use serde_derive::Serialize;

#[derive(Serialize)]
#[serde(variant_identifier)]
enum F {
    A,
    B,
}

fn main() {}
