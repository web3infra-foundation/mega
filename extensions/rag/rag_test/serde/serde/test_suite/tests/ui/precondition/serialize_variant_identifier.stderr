error: variant identifiers cannot be serialized
 --> tests/ui/precondition/serialize_variant_identifier.rs:4:1
  |
4 | / #[serde(variant_identifier)]
5 | | enum F {
6 | |     A,
7 | |     B,
8 | | }
  | |_^
